src/templates/blog-post.tsx

```diff
@@ -98,6 +98,7 @@ export const BlogPostPageQuery = graphql`
               aspectRatio: 1
               layout: FULL_WIDTH
               placeholder: BLURRED
+              transformOptions: { fit: COVER, cropFocus: NORTH }
```

Comment:
@feedm3 , @hofmann-felix Since you are the creators of the pages, please have a look into this 🙏